src/sentry/static/sentry/app/components/forms/apiForm.tsx

```diff
@@ -40,19 +41,36 @@ export default class ApiForm extends Form<Props> {
     this.api.clear();
   }
 
+  groupDataByDisabled() {
+    // Return two hashes, one with data for live elements and another for disabled.
+    const [{data}, disabledData] = [{...this.state}, {}];
+    this.props.children &&
```

Comment:
Would prefer an explicit `if (Array.isArray(this.props.children))` call here instead of the implicit `&&` check. The `Array.isArray` check should also fix the type error below.

Reply:
I like it. Done in fd7f5e346d4cc9589f58853e48815272ae378d01.